R/registry.R

```diff
@@ -9,7 +9,7 @@
 #'
 #' @format An environment.
 #' @keywords internal
-gargle_env <- new.env()
+gargle_env <- new.env(parent = emptyenv())
```

Comment:
Just for my own understanding: this is just good hygiene, to make sure we aren't accidentally picking up entries in the parent?